File: 프로그래머스/unrated/181837. 커피 심부름/커피 심부름.java
class Solution {
    public int solution(String[] order) {
        int answer = 0;
        for(int i = 0; i<order.length; i++){
            if(order[i].charAt(0) == 'a' || order[i].charAt(3) == 'a') answer += 4500;
            else answer += 5000;
        }
        return answer;
    }
}

File: 프로그래머스/0/181837. 커피 심부름/커피 심부름.js
function solution(order) {
    var answer = 0;
    for(let i = 0; i<order.length; i++){
        if(order[i].charAt(0) == 'a' || order[i].charAt(3) == 'a') answer += 4500
        else answer += 5000
    }
    return answer;
}